README.md

```diff
@@ -26,6 +26,7 @@ data into vector tiles that can be rendered dynamically.
 - [OpenLayers 3](https://github.com/openlayers/ol3/pull/4219) - JavaScript vector & raster library.
 - [WhirlyGlobe/Maply](https://github.com/mousebird/WhirlyGlobe/blob/master/WhirlyGlobeSrc/WhirlyGlobe-MaplyComponent/src/MaplyMapnikVectorTiles.mm) - Objective C code that is able to read and render vector tiles(and style with mapnik xml) on iOS devices.
 - [Leaflet.MapboxVectorTile](https://github.com/SpatialServer/Leaflet.MapboxVectorTile) is able to read PBF MapboxVectorTiles from a REST endpoint and render them as a TileLayer on a Leaflet Map. Use this option if you want to utilize vector tiles on a standard Leaflet web map without needing WebGL.
+- [CARTO Mobile SDK](https://github.com/CartoDB/mobile-sdk) - C++-based maps library with focust to offline features, for iOS, Android, Windows Phone and Xamarin with bindings for Java, ObjectiveC and C#. Based on Nutiteq SDK, but Open Source, uses CartoCSS to style MVT vector data.
```

Comment:
* Should this replace the Nutiteq SDK?
* focust to offline features -> offline support